test_unfurl_message.py

```diff
@@ -185,3 +188,170 @@ def test_prepare_description__custom_width():
 )
 def test_get_handler(path_info, handler):
     assert uut.get_handler(path_info) == handler
+
+
+
+@pytest.mark.parametrize(
+    ("name", "api_resp", "want", "want_error"),
+    [
+        (
+                "should return nothing for empty api response",
+                '{}',
+                None,
+                KeyError
+        ),
+        (
+            "happy path",
+            """
+            {
```

Comment:
I'm not gonna block merge based on this, so it's completely up to you, but if you're editing this file anyway, you could extract this to a constant somewhere else, so that the parametrization list stays short, concise, and thus easy to understand.